docs/UserGuide.md

```diff
@@ -245,19 +288,19 @@ Examples:
 * `delete s/A0001234A` deletes a student with a student id of `A0001234A` in the address book.
   ![result for 'find alex', followed by 'delete s/A0001234A'](images/delete.png)
 
-### Finding a common time slot among students: `slots`
+### Finding a common available time slot among students: `slots`
 
 Format : `slots d/DURATION [t/TAG] [p/PERIOD] [o/DAYS]`
 
-* Finds a common slot of time amongst a list of students.
+* Finds a common available slot of time amongst a list of students.
 * The list of students can be narrowed down by tag(s).
-* The period of time to look for the common slot can be specified, otherwise, the default period is between 8 AM to 10 PM.
+* The period of time to look for the available common slot can be specified, otherwise, the default period is between 8 AM to 10 PM.
```

Comment:
I think will be nice to include the specifications here as well, to safeguard ourselves.

```suggestion
* The duration must be specified.
  * Duration is specified with an `int`, it can be between 1 -23, indicating 1-hour to 23-hours, although typically one would use values 1-4/
* The period of time to look for the available common slot can be specified, otherwise, the default period is between 8 AM to 10 PM.
  * The period is specified using 2 `int`s, separated by a `-` (see above). 
* The search on which days can be specified, otherwise, the default will be from Monday to Friday.
  * The days are specified with the lowercase three-letter representation of the day, e.g. `"tue"` or `"wed"`.
```

Reply:
Upvote^ Nice suggestion